server/sonar-server/src/main/java/org/sonar/server/es/IndexerStartupTask.java

```diff
@@ -45,13 +47,14 @@
    * {@link org.sonar.server.issue.index.IssueIndexer}
    */
   public IndexerStartupTask(TestIndexer testIndexer, IssueAuthorizationIndexer issueAuthorizationIndexer, IssueIndexer issueIndexer,
-    UserIndexer userIndexer, ViewIndexer viewIndexer,
+    UserIndexer userIndexer, ViewIndexer viewIndexer, ProjectMeasuresIndexer projectMeasuresIndexer,
```

Comment:
It's the case, but look at the comment higher, some indexer needs to be executed before other one, that's why we can't inject a list